ui/src/components/view/InfoCard.vue

```diff
@@ -834,6 +826,19 @@ export default {
     }
   },
   methods: {
+    updateResourceAdditionalData () {
+      if (!this.resource) return
+      this.resourceType = this.$route.meta.resourceType
+      console.log('Updating resource additional data')
```

Comment:
Please remove this log

Reply:
@utchoang thanks for pointing it out. Done